pkg/external_repos/image_builder.go

```diff
@@ -3,16 +3,16 @@ package external_repos
 import (
 	"encoding/json"
 	"fmt"
-	"io/ioutil"
 	"os"
 	"path/filepath"
 )
 
 // IBUrlsFromDir Import Repo Urls from image builder's git repo
 // The dirPath should point to GITDIR/distributions/ which contains
-// 		the repo directories.  Each directory contains a json file
-//		with the same name as the directory itself:
-//		./distributions/rhel-90/rhel-90.json
+//
+//	the repo directories.  Each directory contains a json file
+//	with the same name as the directory itself:
+//	./distributions/rhel-90/rhel-90.json
```

Comment:
```suggestion
// the repo directories.  Each directory contains a json file
// with the same name as the directory itself:
// ./distributions/rhel-90/rhel-90.json
```

Reply:
so go fmt is doing this.  I turned on 'run go fmt' automatically in my ide, and if you run `go fmt` against a file manually it seems to do this too.  Let me see if i can figure out the reasoning of why its doing this.

Reply:
ah, i see why!